src/Kusto/tools/Resources/.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
The folder **tools** is not required to check-in. Please remove it.

Reply:
done